Sordi christian knowledge

ChristianSordi opened the pull request:
Aggiunta interfaccie e classi per la rete semantica, il commit della creazione de test.txt è stato fatto solo perchè su eclipse era sparito il pacchetto knowledge.

ChristianSordi committed:
Aggiunta Interfaccie

Queste sono le interfaccie necessarie per la rete semantica, quella per il GraphExporter è stato utilizzata per creare un JSON solo per testare la funzionalità della creazione della rete semantica.

ChristianSordi committed:
Correzioni

ChristianSordi committed:
Create Test.txt

ChristianSordi committed:
Aggiunti File necessari Rete Semantica

Aggiunte le classi della rete semantica riorganizzate per le nostre necessità, ovviamente ci saranno degli errori su SemanticNetwork finchè BabelNet non sarà presente.

simonepri commented:
Check [Travis Log](https://travis-ci.org/curiosone-bot/curiosone-core/builds/263478803?utm_source=github_status&utm_medium=notification) and solve errors.

ChristianSordi committed:
Merge branch 'master' into SordiChristian-Knowledge

ChristianSordi committed:
Correzioni Varie

ChristianSordi commented:
Ho corretto alcuni errori del  travis log e fatto commit, si aggiorna da sola questa pull request con i file aggiornati?

simonepri commented:
run `gradle verifyGoogleJavaFormat` from the root of the project in order to automatically fix remaining warns.

ChristianSordi commented:
When i run gradle verifyGoogleJavaFormat from the root of project the result is:
Build Failed with an exception.
Problems: formatting style violations

ChristianSordi committed:
Modifiche GoogleJavaFormat

ChristianSordi committed:
modifiche GoogleJavaFormat

ChristianSordi committed:
modifiche gooleJavaFormat

ChristianSordi committed:
tolte modifiche per GoogleJavaFormat

Ho rimosso le modifiche effettuate per passare il test gradle GoogleJavaFormat poichè sono state inutili...

ChristianSordi committed:
Merge branch 'master' into SordiChristian-Knowledge

dansolombrino commented:
@ChristianSordi why did you reimplement the hashCode function in Concept class? It's already implemented in String  class.
